storage/innobase/srv/srv0start.cc

```diff
@@ -1182,6 +1185,10 @@ static tpool::task rollback_all_recovered_task(trx_rollback_all_recovered,
 					       nullptr,
 					       &rollback_all_recovered_group);
 
+#ifdef UNIV_DEBUG
+bool must_upgrade_ibuf;
+#endif
+
```

Comment:
I could not understand this change. There is no reference to this global. There is one local variable (non-debug) exists by the same name and used. Something to remove ?

Reply:
Right, this should not have been added. I added `ibuf_upgrade_was_needed` instead.